File: Tutorial4-7LayerLibrary/src/main/java/com/test/tutorial4_7layerlibrary/model/ToastMaker.java
package com.test.tutorial4_7layerlibrary.model;

import android.app.Application;
import android.widget.Toast;

public class ToastMaker {

    private Application application;

    public ToastMaker(Application application) {
        this.application = application;
    }

    public void showToast(String message) {
        Toast.makeText(application, message, Toast.LENGTH_SHORT).show();
    }


}

File: Tutorial9-2DynamicFeatureImplementation2/src/main/java/com/smarttoolfactory/tutorial9_2dynamicfeatureimplementation2/model/ToastMaker.kt
package com.smarttoolfactory.tutorial9_2dynamicfeatureimplementation2.model

import android.app.Application
import android.widget.Toast

class ToastMaker(private val context: Application) {
    fun showToast(message: String) {
        Toast.makeText(context, message, Toast.LENGTH_SHORT).show()
    }
}